common/changes/@rushstack/heft/ianc-remove-multiple-tsconfigs_2021-06-15-23-38.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft",
+      "comment": "(BREAKING CHANGE) Remove the ability to run multiple TypeScript compilers in a project.",
```

For anyone potentially impacted, we could give a little more context about why this was eliminated:
```suggestion
      "comment": "(BREAKING CHANGE) Simplify the plugin event hook lifecycle by eliminating an experimental feature that enabled side-by-side compiler configurations. We decided that this scenario is better approached by splitting the files into separate projects.",
```